src/main/java/at/hannibal2/skyhanni/config/features/inventory/BazaarConfig.java

```diff
@@ -49,4 +49,13 @@ public class BazaarConfig {
 
     @Expose
     public Position maxPurseItemsPosition = new Position(346, 90, true, false);
+
+    @Expose
+    @ConfigOption(name = "Craft Materials Bazaar", desc = "Show in a crafting view a shopping list of materials needed when buying from Bazaar.")
```

Comment:
I dont like the description of this but im not sure how to make it better

Reply:
My try (with chatgpts help) of writing the description better.
```suggestion
    @ConfigOption(name = "Craft Materials Bazaar", desc = "In the crafting view, offer a shopping list of required materials for the craft along with a convenient shortcut for purchasing them from the Bazaar.")
```